src/gwt/src/org/rstudio/core/client/VirtualConsole.java

```diff
@@ -276,23 +277,25 @@ private void appendText(String text, String clazz, boolean forceNewRange)
     * @param range
     */
    private void insertText(ClassRange range)
-   {
+   {      
       int start = range.start;
       int end = start + range.length;
-
+      
       Entry<Integer, ClassRange> left = class_.floorEntry(start);
       Entry<Integer, ClassRange> right = class_.floorEntry(end);
-
+      
       // create a view into the map representing the ranges that this class
       // overlaps
       SortedMap<Integer, ClassRange> view = null;
-      if (left != null && right != null)
+      
+      if (left != null && right != null) {
```

Comment:
Formatting, our standard for Java code is opening brace on its own line, and `else` like this:

```java
if (foo)
{
   doSomething();
}
else
{
   somethingElse();
}
```

https://github.com/rstudio/rstudio/wiki/Coding-Standards-(java,-cpp)